### ISSUE TITLE
To implement refresh token method

### ISSUE THREAD

**Issue by guillaumeautret** [kreait/firebase-php#389]
Hi,

In Firebase Auth REST API, i need to use this method: "Exchange a refresh token for an ID token".

For this, i just to add the scope "https://www.googleapis.com/auth/securetoken" in your CONST `API_CLIENT_SCOPES` from Factory Class and implement the new method to use.

```
class Factory
{
    const API_CLIENT_SCOPES = [
        ...
        'https://www.googleapis.com/auth/firebase.remoteconfig',
        'https://www.googleapis.com/auth/userinfo.email',
        'https://www.googleapis.com/auth/securetoken',
    ];
```

Can you implement it, please ?

**Comment by jeromegamez** [comment]
It's already in the making, although there's still much to do! I'll create a PR in the near future and notify you so that you can test it 🤞

**Comment by jeromegamez** [comment]
@guillaumeautret Here you are: https://github.com/kreait/firebase-php/pull/390

**Comment by guillaumeautret** [comment]
Thank you for your response @jeromegamez

**Comment by jeromegamez** [comment]
#390 will take me a couple more days, so I added this as a separate commit to the master branch already (7505e8ade3d7537774ffb93e047bc2d7bed3420d). Feel free to require the `dev-master` branch in the meantime. 🙏

**Comment by jeromegamez** [comment]
Released with [4.41.0](https://github.com/kreait/firebase-php/releases/tag/4.41.0) 🚀
